Makefile

```diff
@@ -233,7 +233,7 @@ start-test-central-repo: stop-test-central-repo ## Starts up a test central repo
 	fi
 	@docker run --rm -d -p 9876:5000 --name central \
 		-v $(ROOT_DIR)/hack/central-repo/registry-content:/var/lib/registry \
-		mirror.gcr.io/library/registry:2 > /dev/null && \
+		mirror.gcr.io/library/registry:2.7.1 > /dev/null && \
```

Comment:
Out of curiosity, why do you prefer to fix the version?  I believe using `:2` gets us the latest as I expect that tag to be updated.

Reply:
This is because CI was failing constantly with this error.
```
Unable to find image 'mirror.gcr.io/library/registry:2' locally
docker: Error response from daemon: manifest for mirror.gcr.io/library/registry:2 not found: manifest unknown: Failed to fetch "2" from request "/v2/library/registry/manifests/2".
See 'docker run --help'.
```

Reply:
If needed I will be fixing this in a separate PR. But as CI was failing constantly, I wanted to test it with this PR.